src/modules/EdgeDetect/EdgeUtils.js

```diff
@@ -118,7 +118,7 @@ function nonMaxSupress(pixels, grads, angles) {
       let angleCategory = categorizeAngle(angles[x][y]);
 
       if (!isOutOfBounds(pixels, x - 1, y - 1) && !isOutOfBounds(pixels, x + 1, y + 1)){
-        switch (angleCategory){
+        switch (angleCategory){ // nom maximum suppression according to angle category
```

Comment:
What is full form of nom? Please use that if it exists

Reply:
It is non lol. So many typos.